src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/FindAndRerankCommand.java

```diff
@@ -36,6 +36,9 @@ public record FindAndRerankCommand(
     @Valid @JsonProperty("sort") FindAndRerankSort sortClause,
     @Valid @Nullable Options options)
     implements ReadCommand, Filterable, Projectable, Windowable {
+  public FindAndRerankCommand {
+    sortClause = (sortClause == null) ? FindAndRerankSort.NO_ARG_SORT : sortClause;
```

Comment:
Uses "compact constructor" to replace missing/null with empty value to avoid NPE later on.